common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -91,7 +95,34 @@ def fetch_file_if_present(filename, fetch_submodules: false)
         raise Dependabot::DependencyFileNotFound, path
       end
 
+      def load_cloned_file_if_present(filename)
+        path = File.join(clone_repo_contents, filename)
+        unless File.exist?(path)
+          raise Dependabot::DependencyFileNotFound, filename
+        end
+
+        content = File.read(path)
+        type = if File.symlink?(path)
+                 symlink_target = File.readlink(path)
+                 "symlink"
+               else
+                 "file"
+               end
+
+        DependencyFile.new(
+          name: Pathname.new(filename).cleanpath.to_path,
```

Comment:
```suggestion
          name: path
```

Reply:
Same as above: I think this should be relative to the repo root, but `path`  that we've assembled is absolute.

Only siths deal in absolutes. :trollface:

Reply:
Ah yes you're right!